### PR TITLE
CLN: Drop compact_ints/use_unsigned from read_csv

### PR DESCRIPTION
**Pull request by gfyoung**
Deprecated in v0.19.0

xref #13323

### CODE REVIEW

**Comment by gfyoung** [comment]
All is green.  Anyone welcome to have a look to merge this.

**Comment by jreback** [comment]
wait on merging anything

**Comment by gfyoung** [comment]
Ah, right.  Travis is still failing.  Got it.

File: pandas/io/parsers.py [hunk 1]
```diff
@@ -1569,11 +1544,6 @@ def _convert_to_ndarrays(self, dct, na_values, na_fvalues, verbose=False,
                 if cast_type and not is_dtype_equal(cvals, cast_type):
                     cvals = self._cast_types(cvals, cast_type, c)
 
-            if issubclass(cvals.dtype.type, np.integer) and self.compact_ints:
```
Comment (jreback) [inline pandas/io/parsers.py]: I believe we can also blow away ``lib.downcast_int64``, ``to_numeric`` uses a different (and better) mechanism for doing this. IIRC ``downcast_int64`` was only for compatibility.

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/io/parsers.py (hunk 1 above)
Comment (gfyoung) [reply pandas/io/parsers.py]: GitHub search seems to verify that belief.

**Comment by gfyoung** [comment]
Function removed, and all is green still.  PTAL.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @gfyoung

**Pull request merged**